### PR TITLE
Redis service tests

### PR DESCRIPTION
**Pull request by Dire-0x**
Tests for all the public methods in the Redis Service. Tests all current transforms for events and Redis data.

### CODE REVIEW

**Comment by Dire-0x** [comment]
Not sure why the checks fail at `npm ci`

File: tsconfig.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "compilerOptions": {
-    "target": "es2017",
+    "target": "es2020",
```
[the remark this answers is not in the document]
Comment (Dire-0x) [reply tsconfig.json:3]: VS code was complaining about the use of bigint with target of es2017 which became apparent in testing

[the remark this answers is not in the document]
Comment (henridf) [reply tsconfig.json:3]: Ok, please can you remove node 12 from the actions yml file then.

[the remark this answers is not in the document]
Comment (henridf) [reply tsconfig.json:3]: > Ok, please can you remove node 12 from the actions yml file then.

Actually we can do that in a separate PR if you prefer... README also needs to be updated to specify min node version too at that point.

**Review by henridf — APPROVED** [review]

**Pull request merged**